### PR TITLE
Update device guidance page

### PR DESCRIPTION
**Pull request by aliuk2012**
### Changes proposed in this pull request
- Updates the device guidance page design and copy
- Removes next/previous links at the bottom of the guidance pages
- Refactor how the device guidance page is built using key/value from en.yml

### Guidance to review

Visit https://dfe-ghwt-pr-1498.herokuapp.com/devices and view a couple of the links

### CODE REVIEW

**Comment by aliuk2012** [comment]
[Update the Device guidance pages index page (ignore urls for now)](https://trello.com/c/Huol5XoW/1738-update-the-device-guidance-pages-index-page-ignore-urls-for-now)

File: app/views/devices_guidance/index.html.erb [hunk 1]
```diff
@@ -12,14 +12,41 @@
       <%= t('landing_pages.get_support_guides.title' ) %>
     </h1>
 
-    <p class="govuk-body">Many vulnerable and disadvantaged children and young people in England do not have access to a laptop, tablet or internet connection. The Department for Education (DfE) is offering support to help children, young people and families access remote education and social care during the pandemic.</p>
-
     <p class="govuk-body">
-      The DfE is working with delivery partners to supply local authorities, academy trusts and schools
-      with <%= govuk_link_to 'laptops and connectivity packages', 'https://www.gov.uk/guidance/get-help-with-technology-for-remote-education-during-coronavirus-covid-19' %>.
+      How to choose, order, and safely set up laptops and tablets for pupils in <span class="app-no-wrap">years 3 to 13</span>,
```
Comment (aliuk2012) [inline app/views/devices_guidance/index.html.erb:16]: I agree

Comment (aliuk2012) [inline app/views/devices_guidance/index.html.erb:16]:
```suggestion
      How to choose, order, and safely set up laptops and tablets for pupils in years 3 to 13,
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -272,25 +272,52 @@ en:
...
       description: Find out how we estimated device requirements and how to query an allocation.
       audience: responsible_body_users
-    how_to_order:
-      title: How and when to order laptops
-      description: How to order laptops for disadvantaged children and young people during coronavirus (COVID-19).
-      audience: responsible_body_users
+#    how_to_order:
+#      title: How and when to order laptops
+#      description: How to order laptops for disadvantaged children and young people during coronavirus (COVID-19).
+#      audience: responsible_body_users
```
Comment (aliuk2012) [inline config/locales/en.yml:285]: Should not be a comment? possibly remove?

File: spec/models/multipart_guidance_spec.rb [hunk 3]
```diff
@@ -82,9 +82,9 @@
 
   describe '#pages_for' do
     it 'filters the pages by audience' do
```
Comment (aldavidson) [inline spec/models/multipart_guidance_spec.rb:84]: If it's not using `audience:` anymore, can this example description be changed to reflect that? How about:

```suggestion
    it 'filters the pages by guidance_section' do
```

**Review by aldavidson — APPROVED** [review]
Looks good, just one suggestion about updating the name of the spec

**Pull request merged**